MapleServer2/Commands/Game/ItemCommands.cs

```diff
@@ -37,6 +37,7 @@ public override void Execute(GameCommandTrigger trigger)
         Item item = new(itemId)
         {
             Rarity = rarity >= 0 ? rarity : ItemMetadataStorage.GetRarity(itemId),
+            TransferFlag = ItemMetadataStorage.GetTransferFlag(itemId, rarity),
```

Comment:
Doesn't this already happens with **SetMetadataValues** on item creation?

Reply:
The command is technically scuffed last time we talked about it.
The rarity is not displayed properly until after you change maps/relog. (try spawning an item and chat linking it)
And with rarity not correctly updating till then, it also affects the transfer flags for this.